backend/btrixcloud/storages.py

```diff
@@ -24,462 +25,497 @@
     sync_get_log_stream,
 )
 
+from .utils import is_bool
+
 
 CHUNK_SIZE = 1024 * 256
 
 
 # ============================================================================
-def init_storages_api(org_ops, crawl_manager, user_dep):
-    """API for updating storage for an org"""
+class StorageOps:
+    """All storage handling, download/upload operations"""
 
-    router = org_ops.router
-    org_owner_dep = org_ops.org_owner_dep
+    def __init__(self, org_ops, crawl_manager):
+        self.storages = {}
+
+        self.org_ops = org_ops
+        self.crawl_manager = crawl_manager
+
+        self.is_local_minio = is_bool(os.environ.get("IS_LOCAL_MINIO"))
+
+        with open("/tmp/storages/storages.json", encoding="utf-8") as fh:
```

Comment:
The `storages.json` is just the list in the storages field, which by default is:

```
storages:
  - name: "default"
    access_key: "ADMIN"
    secret_key: "PASSW0RD"
    bucket_name: *local_bucket_name

    endpoint_url: "http://local-minio.default:9000/"
```

I'm not sure it makes sense splitting this into a separate file from values.yaml, and for other settings, we just make these env vars, but made this a separate file in case it can get long (but could put the JSON value as an env var too).

Reply:
Ah okay, I see! Yeah that should work well.